php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -170,7 +170,7 @@ class NDB_BVL_Instrument extends NDB_Page
      */
     static function factory($instrument, $commentID, $page, $guarantee_exists = true)
     {
-        //include_once 'HTML/QuickForm.php';
+        include_once 'HTML/QuickForm.php';
```

Comment:
Do the instrument_manager problems you were having happen if you manually bear install HTML_Quickform without the include?